r2/r2/templates/base.html

```diff
@@ -30,7 +30,8 @@
   <head>
     <title>${self.Title()}</title>
 
-    <meta name="viewport" content="initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"/>
+<!-- `meta name="viewport"` set for mobile browsers. See https://github.com/tog22/eaforum/issues/7 & https://github.com/tog22/eaforum/issues/28 -->
```

Comment:
OK will do, though we have found HTML comments to work decently for our decentralised volunteer team.